charts/project-origin-registry/templates/registry-deployment.yaml

```diff
@@ -31,10 +31,27 @@ spec:
           args:
             - "--serve"
           env:
+          # General configuration
             - name: RegistryName
               value: {{ .Values.registryName | default .Release.Name }}
+
+          # TransactionProcessor configuration
+            - name: TransactionProcessor__ServerNumber
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['apps.kubernetes.io/pod-index']
+            - name: TransactionProcessor__Servers
+              value: {{ .Values.transactionProcessor.replicas | quote }}
+            - name: TransactionProcessor__Threads
+              value: {{ .Values.transactionProcessor.threads | quote }}
+            - name: TransactionProcessor__Weight
+              value: {{ .Values.transactionProcessor.weight | quote }}
```

Comment:
How are the values calculated, or determined? Manually?

Reply:
They come from the values.yaml file, they are determined by the ones deploying the registry.